packages/menu-button/src/index.tsx

```diff
@@ -662,10 +662,10 @@ const MenuItems = React.forwardRef(function MenuItems(
           // consistent behavior across menu items we'll trigger a click when
           // the spacebar is pressed.
           if (selected) {
+            event.preventDefault();
```

Comment:
@IanVS Should this not be moved into the next `if` block? I don't think we want to `preventDefault` for buttons.

Reply:
Ah nope, I'm totally wrong here since we were doing that already! Gonna poke around a bit for any unintended edge cases here.